README.md

```diff
@@ -1,6 +1,6 @@
-# OAK-docs
+# Ava Protocol-docs
```

Comment:
Change to

# Ava Protocol Documentation

Reply:
Fixed